typescript/end-to-end-testing/test/test.ts

```diff
@@ -0,0 +1,142 @@
+import * as restate from "@restatedev/restate-sdk";
+
+import {
+  GenericContainer,
+  StartedTestContainer,
+  TestContainers,
+  Wait,
+} from "testcontainers";
+import { createPromiseClient, Transport } from "@connectrpc/connect";
+import { createConnectTransport } from "@connectrpc/connect-node";
+import { ExampleService } from "./generated/example_connect";
+import * as http2 from "http2";
+import * as net from "net";
+
+import { protoMetadata } from "../src/generated/proto/example";
+import { MyExampleService } from "../src/example_service";
+
+// Prepare the restate server
+async function prepareRestateServer(
```

Comment:
Can we pull these out into another file so users could basically copy over the file and write their own tests?

Reply:
Can do